docs/src/man/other_operators.md

```diff
@@ -64,6 +64,16 @@ where ``\mathbf{I}`` is the second order identitiy tensor.
 inv
 ```
 
+## Backslash
+
+Left division operator such that ``\mathbf{A} \backslash \mathbf{b} = \mathbf{A}^{-1} \cdot \mathbf{b}``.
```

Comment:
if we support `Tensor{2} \ Tensor{2}` then perhaps we should also allow `Tensor{2} \ Tensor{4}` ?